python/tests/hypothesis/arcticdb/test_hypothesis_version_store.py

```diff
@@ -114,7 +114,7 @@ def _get_latest_undeleted_version(self, sym) -> Tuple[Optional[int], Optional[Ve
 
     @rule(
         sym=symbols,
-        prune=st.sampled_from([True, False]),  # booleans() shrinks towards False, but prune is on by default
+        prune=st.sampled_from([True, False]),  # booleans() shrinks towards False
```

Comment:
This change is incorrect: most tests don't use `library.py` so will respect the library option

Reply:
do you mean that they might use a library where prune_previous_versions option is explicitly set? @qc00

Reply:
Each library option has a default value which is used if not explicitly overridden when called in Python.

The `library.py` methods have your `prune_previous_version=False` changes, but they're not used in this test or most other tests. They use the `_store.py` methods, which has `prune_previous_version=None`.

Reply:
Understood. However, this commit only involves deleting the 'but prune is off by default' comment. With the default value of the library's prune_previous_version attribute and the default value of the function call for the same attribute, the comment can still be confusing. So, I think is good to remove it, as @poodlewars suggested

Reply:
Docs should be the way to disambiguate any question/doubt. The comment should relate only to the test itself

Reply:
But if you remove that bit, the whole comment makes no sense.

The comment indicated the unusual behaviour of using `st.sampled_from([True, False])` instead of `st.booleans()` is because the latter shrinks towards the default library option, which has less testing power than the non-default (`True`).

Reply:
As we're manually passing in the keyword arg, and `but prune is on by default` refers to the library option which is being overridden, I think removing this part of the comment makes things clearer.

Reply:
Yeah the comment makes sense without the last clause. It's saying that we have that line to force a shrink towards True. The default never affected the actual logic of the test.

Reply:
agree